### PR TITLE
[#5] Add 5 jokes

### PR DESCRIPTION
**Pull request by eddyannobil**
Went online to fetch 5 jokes to add to the jokes.md file

### CODE REVIEW

File: jokes.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+1. A pair of cows were talking in the field. One says, “Have you heard about the mad cow disease that’s going around?”
+
```
Comment (fiifikrampah) [inline jokes.md:2]: I think we can delete the spaces between the joke and their "responses" right? It sort of gives the impression that the response is a separate joke on its own.

**Review by fiifikrampah — CHANGES_REQUESTED** [review]
@eddyannobil I think this is in pretty good shape. Address this change and then we'll take it from there.

**Review by enock500 — COMMENTED** [review]
@eddyannobil please check out these changes

**Review by enock500 — CHANGES_REQUESTED** [review]
@eddyannobil please check your PR

File: jokes.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+“Yeah,” the other cow says. “Makes me glad I’m a penguin.”
+
+2. “Your mother has been with us for 20 years,” said John. “Isn’t it time she got a place of her own?”
+
+“My mother?” replied Helen. “I thought she was your mother.”
+
+3. “Once, my father came home and found me in front of a roaring fire. That made my father very mad, as we didn’t have a fireplace.” —Victor Borge
+
+4. Why don’t pirates take a shower before they walk the plank?
+
```
Comment (enock500) [inline jokes.md]: Please clear all white spaces

**Review by enock500 — CHANGES_REQUESTED** [review]
@eddyannobil PR looks good

File: jokes.md (hunk 1 above)
Comment (eddyannobil) [reply jokes.md:2]: Which spaces? The spaces between the words or the spaces between the sentences?

Comment (fiifikrampah) [reply jokes.md:2]: The spaces b/n the sentences. Each joke is in a call-response format right? So we don't need an extra space between the call and response.

File: jokes.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+1. A pair of cows were talking in the field. One says, “Have you heard about the mad cow disease that’s going around?”
```
Comment (fiifikrampah) [inline jokes.md]: @eddyannobil  So something like this: 
```suggestion
1. A pair of cows were talking in the field. One says, “Have you heard about the mad cow disease that’s going around?”
“Yeah,” the other cow says. “Makes me glad I’m a penguin.”
```

**Review by fiifikrampah — APPROVED** [review]
Looks like you have a slight typo in the commit message. Please fix it.

**Pull request merged**